src/site/content/en/blog/compress-images-avif/index.md

```diff
@@ -123,7 +123,7 @@ brings all the libaom improvements mentioned in the above.
 Note: [libheif](https://github.com/strukturag/libheif)
 is another popular open source AVIF muxer and parser,
 used in
-[ImageMagick](https://imagemagick.org/index.php),
```

Comment:
This doesn't seem broken. You can maybe strip the trailing "index.php" and just link to the domain itself.